core/src/test/java/com/scalar/db/storage/jdbc/JdbcTableMetadataManagerTest.java

```diff
@@ -0,0 +1,533 @@
+package com.scalar.db.storage.jdbc;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doAnswer;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.scalar.db.api.Scan.Ordering.Order;
+import com.scalar.db.api.TableMetadata;
+import com.scalar.db.io.DataType;
+import com.scalar.db.storage.jdbc.JdbcTableMetadataManagerTest.ResultSetMocker.Row;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import javax.sql.DataSource;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.invocation.InvocationOnMock;
+
+public class JdbcTableMetadataManagerTest {
```

Comment:
It looks like the spotbugs report shows some false positives:
https://4146-151700458-gh.circle-artifacts.com/0/gradle_spotbugs_reports_for_core/spotbugs/test.html

I think we can suppress the warnings with `@SuppressFBWarnings("OBL_UNSATISFIED_OBLIGATION")`.

Reply:
Indeed, since the objects are mocked, it's a false positive